### DISCUSSION TITLE
change_property()

### DISCUSSION THREAD

**Discussion by rickygill00** [ansys/pyaedt#4781] [Q&A] [answered]
For the example "Maxwell 2D: PM synchronous motor transient analysis" [link](https://aedt.docs.pyansys.com/version/stable/examples/03-Maxwell/Maxwell2D_PMSynchronousMotor.html#sphx-glr-examples-03-maxwell-maxwell2d-pmsynchronousmotor-py)

How do you use change_property() to change the Stator: name, colour, material etc. I couldn't find any examples in the pyaedt documentation.

**Comment by fwlfHS** [comment, marked as answer]
Hello, not sure if you need the change_property() function. you can change the assignment by using the methods given in the [example](https://aedt.docs.pyansys.com/version/stable/examples/03-Maxwell/Maxwell2D_PMSynchronousMotor.html#assign-properties-to-stator)
However if you use [change_property](https://aedt.docs.pyansys.com/version/0.9/API/_autosummary/pyaedt.maxwell.Maxwell2d.change_property.html#change-property) it's best to use the record macro feature in the GUI to get the tab names and arguments that you need. However I would only use that method if there is no simpler way of changing properties by direct assignment through the API
